### DISCUSSION TITLE
How do check port is is opened

### DISCUSSION THREAD

**Discussion by MohMehrnia** [jchristn/WatsonTcp#211] [General]
How do check port is is opened?

**Comment by jchristn** [comment]
Hi @MohMehrnia this isn't a function of the library.  It will throw an exception if the port is in use.  There are tools for each operating system that will tell you whether or not a port is in use (generally it's ```netstat```).

Windows: https://stackoverflow.com/questions/273159/how-do-i-determine-if-a-port-is-open-on-a-windows-server
Mac: https://apple.stackexchange.com/questions/117644/how-can-i-list-my-open-network-ports-with-netstat
Linux: https://serverfault.com/questions/309052/check-if-port-is-open-or-closed-on-a-linux-server
